### DISCUSSION TITLE
How to add (boost headers module) only in the manifest file

### DISCUSSION THREAD

**Discussion by AbdulAbCellera** [microsoft/vcpkg#21208] [Q&A] [answered]
I want to use `find_package(Boost 1.77.0 EXACT REQUIRED)` in a cmake file, but not sure what is the correct module to add in my vcpkg.json. 

I am thinking to add `"dependencies": [
        {
            "name": "boost",
            "version>=": "1.77.0"
        }]
` but according to https://vcpkg.info/ the boost module has many build dependencies, so it might be slow.

any suggestions?

**Comment by mathisloge** [comment, marked as answer]
just install those libs you need. with `vcpkg search boost` you can list all boost libraries. 
e.g. if you need boost signals2 (which is a header only lib), your dep is `boost-signals2`
